builtin/client/chatcommands.lua

```diff
@@ -2,6 +2,10 @@
 
 
 core.register_on_sending_chat_messages(function(message)
+	if message:sub(1,2) == ".." then
```

Comment:
Can I no longer make a command like in worldedit where you have `//`? Maybe escape with a backslash?